src/lib/index.ts

```diff
@@ -143,24 +143,24 @@ export class GoogleToken {
    * @param callback The callback function.
    */
   public revokeToken(callback: (err?: Error) => void): void {
-    if (this.token) {
-      request(GOOGLE_REVOKE_TOKEN_URL + this.token, (err: Error) => {
-        if (err) {
-          callback(err);
-          return;
-        }
-        this.configure({
-          email: this.iss,
-          sub: this.sub,
-          key: this.key,
-          keyFile: this.keyFile,
-          scope: this.scope
-        });
-        callback();
-      });
-    } else {
+    if (!this.token) {
       process.nextTick(callback, new Error('No token to revoke.'));
+      return;
     }
+    axios.get(GOOGLE_REVOKE_TOKEN_URL + this.token)
+        .then(r => {
```

Comment:
It would be good to use async/await rather than .then/.catch However I am not sure if that is semver major. Thoughts?

Reply:
I completely agree with you, which is why I filed #25 :P  Lets hit it in the next CL.